misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/util/ConverterUtils.java

```diff
@@ -0,0 +1,178 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.converters.util;
+
+import io.ballerina.compiler.syntax.tree.Node;
+import io.ballerina.compiler.syntax.tree.NodeList;
+import io.ballerina.compiler.syntax.tree.RecordFieldNode;
+import io.ballerina.compiler.syntax.tree.RecordTypeDescriptorNode;
+import io.ballerina.compiler.syntax.tree.TypeDefinitionNode;
+import io.ballerina.converters.exception.ConverterException;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+/**
+ * Util methods for JSON to record converter.
+ */
+public class ConverterUtils {
+    /**
+     * Method for mapping openApi type to ballerina type.
+     * @param type  OpenApi parameter types
+     * @return ballerina type
+     */
+    public static String convertOpenAPITypeToBallerina(String type) {
+        String convertedType;
+        switch (type) {
+            case Constants.INTEGER:
+                convertedType = "int";
+                break;
+            case Constants.STRING:
+                convertedType = "string";
+                break;
+            case Constants.BOOLEAN:
+                convertedType = "boolean";
+                break;
+            case Constants.ARRAY:
+                convertedType = "[]";
+                break;
+            case Constants.OBJECT:
+                convertedType = "record";
+                break;
+            case Constants.DECIMAL:
+            case Constants.NUMBER:
+            case Constants.DOUBLE:
+                convertedType = "decimal";
+                break;
+            case Constants.FLOAT:
+                convertedType = "float";
+                break;
+            default:
+                convertedType = "anydata";
+        }
+        return convertedType;
+    }
+
+    /**
+     * This method will escape special characters used in method names and identifiers.
+     *
+     * @param identifier - identifier or method name
+     * @return - escaped string
+     */
+    public static String escapeIdentifier(String identifier) {
+
+        if (identifier.matches("\\b[0-9]*\\b")) {
+            return "'" + identifier;
+        } else if (!identifier.matches("\\b[_a-zA-Z][_a-zA-Z0-9]*\\b"
+        ) || Constants.BAL_KEYWORDS.stream().anyMatch(identifier::equals)) {
```

Comment:
This starting bracket should be in the earlier line.

Reply:
Addressed with 5c09fd7